.github/workflows/linux-eic-shell.yml

```
@@ -1233,3 +1235,16 @@ jobs:
           issue-number: ${{ github.event.pull_request.number }}
           body-file: capybara_${{ github.event.pull_request.number }}.md
           edit-mode: replace
+
+  cancel-container:
+    if: cancelled() || failure()
```

```suggestion
    if: cancelled()
```
All relevant failures will not trigger container in the first place. All irrelevant ones do not matter.

---

`trigger-container` only depends on `eicrecon-gun`. If e.g. `eicrecon-dis` or so fails, then we want the container pipeline to be canceled as well.

---

No contradiction here. If it was not in dependencies, it didn't matter enough to be added.